src/Connect.ts

```diff
@@ -6,6 +6,44 @@
 import * as Util from './Util';
 import { NetAddress } from './NetAddress';
 
+/**
+ * Parameters of a key system for encrypted streams (DRM)
+ *
+ * If the key system is a string, it's the URL of the license server.
+ *
+ * If the key system is an object, it's a key system configuration with more parameters.
+ */
+export type KeySystem =
+    | string
```

Comment:
why this string? it's not always a structure ?

Reply:
As described in the common to make the use faster you can put a string if there is only a licenseUrl parameter, as other players do. Like this : 

```
{
  'com.widevine.alpha' : {
    licenseUrl: 'https://licensewidevine/url',
    audioRobustness: [''],
    videoRobustness: ['SW_SECURE_DECODE']',
  },
  'com.microsoft.playready' : 'https://licenseplayready/url',
}
```